### PR TITLE
Devel ergocub sn002

### PR DESCRIPTION
**Pull request by pattacini**
Alignment in view of distro 2024.11.0.

### CODE REVIEW

**Comment by pattacini** [comment]
cc @Nicogene

File: ergoCubSN002/calibrators/left_arm-calib.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
 	</group>
 
 <!--	<param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8) (9) (10 11 12) </param> --> <!-- Don't remove this line -->
-	<param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8) (9) (10 11 12) </param> 
+	<param name="CALIB_ORDER"> (3) (2) (0) (1) (7 8) (10 11 12) </param>
```
Comment (pattacini) [inline ergoCubSN002/calibrators/left_arm-calib.xml:34]: Point to discuss.

File: ergoCubSN002/calibrators/right_arm-calib.xml [hunk 2]
```diff
@@ -32,7 +32,8 @@
 	</group>
 
 <!--	<param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8) (9) (10 11 12) </param> --> <!-- Don't remove this line -->
-	<param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8) (9) (10 11 12) </param> 
+        <!-- <param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8) (10 11 12) </param> -->
+	<param name="CALIB_ORDER"> (3) (2) (0) (1) (7 8) (10 11 12) </param>
```
Comment (pattacini) [inline ergoCubSN002/calibrators/right_arm-calib.xml:36]: Same here.

File: ergoCubSN002/hardware/mechanicals/left_leg-eb9-j4_5-mec.xml [hunk 3]
```diff
@@ -30,7 +30,7 @@
         <param name="AutoCalibration">           0             0   </param>
         <param name="Verbose">                   0             0   </param>
         <param name="HasHallSensor">             0             0   </param>
-        <param name="TemperatureSensorType">    PT100        PT100 </param> <!-- Supported values: PT100 or NONE. If setting NONE consider using the safer current limits for the ankel roll. -->
+        <param name="TemperatureSensorType">    PT100         NONE </param> <!-- Supported values: PT100 or NONE. If setting NONE consider using the safer current limits for the ankel roll. -->
```
Comment (pattacini) [inline ergoCubSN002/hardware/mechanicals/left_leg-eb9-j4_5-mec.xml:33]: Point to discuss.

**Review by pattacini — COMMENTED** [review]
Some FT are disabled.
cc @S-Dafarra @GiulioRomualdi

**Comment by S-Dafarra** [comment]
The robot just went into maintenance to fix the wrists, the neck, and (in theory) also the broken temperature sensor on the ankle

**Comment by pattacini** [comment]
> The robot just went into maintenance to fix the wrists, the neck, and (in theory) also the broken temperature sensor on the ankle

Same as https://github.com/robotology/robots-configuration/pull/691#discussion_r1862051848.

**Comment by traversaro** [comment]
Sorry, I accidentally solved the conflict of the PR, not sure if this is done according the workflow. Anyhow, today with @S-Dafarra  we were able to run the robot, and even if we only did the teleoperation test with a mixed YARP 3.10/YARP 3.9 setup. So I think at this point we can merge this and release 2024.11.0, what do you think @Nicogene @pattacini ?

**Comment by pattacini** [comment]
Great! I'll merge the PR and tag the repo.

**Comment by pattacini** [comment]
Apparentely, I cannot rebase & merge[^1] because of

image

[^1]: We ought to rebase & merge not to lose intermediate commits but to keep history a bit cleaner.

**Comment by pattacini** [comment]
Hi @traversaro 

Is this fork/branch already rebased on top of upstream devel?

**Comment by traversaro** [comment]
> Hi @traversaro
> 
> Is this fork/branch already rebased on top of upstream devel?

I am afraid not, I accidentally did a conflict resolution from the GitHub UI (that we can drop), but I did not interacted with the branch in any other way.

**Comment by pattacini** [comment]
It's not rebased. Although the conflicts are solved, GitHub is unable to rebase & merge.
I'll try to rebase it.

**Pull request merged**